etc/ESSR/R/.load.R

```diff
@@ -30,14 +30,25 @@
     VERSION <- "1.7"
     assign(".ess.ESSRversion", VERSION, envir = ESSR)
 
+    ## either source the ESSR files if a directory is provided, or otherwise try
+    ## to move any existing R objects into the ESSR environment
+    if(!is.null(dir)) {
```

Comment:
It's a bit confusing as two distinct functionalities are packed in one function, especially if the name of the function does not represent what it does. 

What I meant with my comment is to extract the moving-to-ESSR-enviornment into a function. Place it in this file along side .ess.load.ESSR.  Maybe `.ess.collect.ESSR.objects` (should contain dots only to be compatible with the pre-historic versions of R).

Reply:
I've added some commits that split out the parts of `.ess.load.ESSR` that are needed for reuse in the new `.ess.collect.ESSR.objects` function, and then create the new function.